src/consts/env_defaults.py

```diff
@@ -4,9 +4,9 @@
 
 DEFAULT_NUMBER_OF_MASTERS: int = consts.NUMBER_OF_MASTERS
 DEFAULT_DAY2_WORKERS_COUNT: int = 1
-DEFAULT_WORKERS_COUNT: int = 0
+DEFAULT_WORKERS_COUNT: int = 2
```

Comment:
seems like you're changing the default that was until now, of using a "standalone" (3 nodes) clusters
I think it's better to revert it, so that it will not impact local environments that used to have enough resources for a basic installation in test-infra

Reply:
The default on CI was 2, and it already changes some of the behavior on CI (https://github.com/openshift/release/pull/25792)
 I just moved it to test-infra

Reply:
okay, so we glitched back then
I guess if no one complained then it's okay